src/graphql/client.js

```diff
@@ -0,0 +1,59 @@
+import { createClient, dedupExchange, cacheExchange, fetchExchange } from 'urql';
+import { authExchange } from '@urql/exchange-auth';
+import { makeOperation } from '@urql/core';
+import { SK_PAT, SK_USER } from '../constants';
+import { readFromLocalStorage } from '../api/chrome';
+
+const getAuth = async ({ authState }) => {
+  if (!authState) {
+      const { pat } = await readFromLocalStorage([SK_PAT, SK_USER])
+
+      if (!pat) {
+          throw Error('PAT not available.') 
+      }
+
+      return { token: pat }
+  }
+
+  return null;
+};
+
+const addAuthToOperation = ({ authState, operation }) => {
+  if (!authState || !authState.token) {
```

Comment:
minor: we can probably use conditional chaining operator here

Reply:
Yepp. Done